packages/apps/webhooks/src/events-to-webhooks-app.ts

```diff
@@ -1,5 +1,7 @@
 import type { ApiEventHandler } from '@cloudcommerce/firebase/lib/helpers/pubsub';
-import type { Carts, Customers, Orders } from '@cloudcommerce/api/types';
+import type {
+  Carts, Customers, Orders, ResourceId,
+} from '@cloudcommerce/api/types';
```

Comment:
```suggestion
import type {
  ResourceId,
  Carts,
  Customers,
  Orders,
} from '@cloudcommerce/api/types';
```

ESLint tá funcionando na sua IDE? 🤔